src/getrepos.ts

```diff
@@ -12,21 +13,23 @@ export async function getRepoIds(repoPath: string, clientInput: {account, access
       `Find CodeRepo with name='${repo}'`,
     );
     if (repoID.length === 0) {
```

Comment:
Can you flatten out this branch structure? In its current form it's a bit confusing to read.

For example, instead of this:
```
if (condition) {
  if (otherCondition) {
    doThing();
  } else {
    doOtherThing();
  }
} else {
  theBeatles();
}
```

You could have:

```
if (!condition) {
  theBeatles();
  continue;
}

if (!otherCondition) {
  doOtherThing();
  continue;
}

doThing();
```